### PR TITLE
feat: Algolia auto trigger instead of manual trigger

### PR DESCRIPTION
**Pull request by calldelegation**

### CODE REVIEW

**Review by Nazeeh21 — APPROVED** [review]
Looks good to me

File: .github/workflows/docs-scrapper.yml [hunk 1]
```diff
@@ -3,11 +3,14 @@ name: DocSearch Scrap
...
   # Allow to run manually
   workflow_dispatch:
+  # Trigger on successful deployment status
+  deployment_status:
 
 jobs:
   update-docsearch:
     name: Update DocSearch
     runs-on: ubuntu-latest
+    if: github.event_name == 'deployment_status' && github.event.deployment_status.state == 'success' && github.ref == 'refs/heads/master'
```
Comment (petertonysmith94) [inline .github/workflows/docs-scrapper.yml:13]: Do we want to be able to trigger this manually?

```suggestion
    if: github.event_name == 'workflow_dispatch' || (github.event_name == 'deployment_status' && github.event.deployment_status.state == 'success' && github.ref == 'refs/heads/master')
```

**Review by petertonysmith94 — COMMENTED** [review]
Approved with comment 🔥

**Review by petertonysmith94 — APPROVED** [review]

**Review by Nazeeh21 — APPROVED** [review]
Approved

**Pull request merged**